package.json

```diff
@@ -17,6 +17,7 @@
   "files": [
     "dist/",
     "loader/",
+    "react/",
```

Comment:
I do `import GdsButton from 'genero-design-system/react'` 

but i get:
> Module not found: Can't resolve 'genero-design-system/react'

Reply:
Yeah because it's not published to npm.

You can use [`npm link`](https://docs.npmjs.com/cli/link.html) to link the package to your local copy while developing.

Reply:
Would be great if you could document `npm link` as well!

Reply:
after `npm link` react-scripts still fails to compile

Reply:
i also tried to install with the path so that this was in my package.json `"genero-design-system": "file:../../dev/genero-design-system"`

Reply:
oh right.. the typescript probably needs to be compiled first 😅